services/__tests__/__snapshots__/mail.test.js.snap

```diff
@@ -3,11 +3,9 @@
 exports[`buildMailOptions build mail options for a html email 1`] = `
```

Comment:
This is from upgrading `jest-serializer-html` which trims the output more than before.